tests/calls.rs

```diff
@@ -12,46 +13,46 @@ fn has_called() {
 	ext_reset(|e| e);
 	let mut result = [0u8; 1];
 	let input = [2u8; 32];
-	pwasm_ethereum::call(2000, &Address::new(), 10000.into(), &input, &mut result).expect_err("Should be an Error");
+	pwasm_ethereum::call(2000, &Address::zero(), 10000.into(), &input, &mut result).expect_err("Should be an Error");
```

Comment:
I know this boat already sailed, but personally I think `Address::new()` reads better than `Address::zero()`. :/

Reply:
I am not even the person that declared it being deprecated in version `0.2` of `fixed-hash`, however, I am still in favor of this change since it is more clear what the value of the constructed `fixed-hash` will be.